packages/hebao_v1/contracts/base/MetaTxModule.sol

```diff
@@ -235,10 +236,12 @@ abstract contract MetaTxModule is BaseModule
     /// @param wallet The wallet used in all transactions.
     /// @param signers The signers needed for all transactions.
```

Comment:
signers The signers needed for all transactions. =>
signers The signers needed for all transactions whose signatures have been verified.